docs/source/refs.bib

```diff
@@ -51,18 +65,19 @@ @inproceedings{DBLP:conf/qce/BurgholzerBW21
   pages     = {199--206},
   publisher = {{IEEE}},
   year      = {2021},
-  url       = {https://doi.org/10.1109/QCE52317.2021.00037},
   doi       = {10.1109/QCE52317.2021.00037},
   timestamp = {Tue, 30 Nov 2021 17:31:17 +0100},
   biburl    = {https://dblp.org/rec/conf/qce/BurgholzerBW21.bib},
-  bibsource = {dblp computer science bibliography, https://dblp.org}
+  bibsource = {dblp computer science bibliography, https://dblp.org},
+  url       = {https://www.cda.cit.tum.de/files/eda/2021_qce_hybrid_schrodinger_feynman_simulation_with_decision_diagrams.pdf}
 }
 
 @inproceedings{burgholzer2022simulation,
       title={Simulation Paths for Quantum Circuit Simulation with Decision Diagrams},
       author={Lukas Burgholzer and Alexander Ploier and Robert Wille},
       year={2022},
       booktitle = {Design, Automation {\&} Test in Europe Conference},
```

Comment:
This bib-entry is slightly wrong as it refers to the DATE short paper, although it should refer to the TCAD journal extension.